README.md

```diff
@@ -16,20 +16,49 @@ python3 setup.py install --user
 ```
 pip3 install standup-notes
 ```
-
-## Run
+###Bash Setup
+For a 1 time execution, use
+```
+source /path/to/standup-notes.bash
+```
+To permanently have bash completion, copy the standup-notes.bash into the bash-completion.d folder in your system
 ```
+cp /path/to/standup-notes.bash /path/to/bash_completion.d
+```
+
+## Commands
+#####List
+ ```
+standup-notes --l
 standup-notes --list
-standup-notes --read-yesterday
-standup-notes --read-today
-standup-notes --read-tomorrow
-standup-notes --edit-yesterday
-standup-notes --edit-today
-standup-notes --edit-tomorrow
-standup-notes --copy-yesterday
-standup-notes --copy-today
-standup-notes --copy-tomorrow
+ ``` 
+ will list commands that can be run by the script
+####Read
```

Comment:
Are there instructions on specifying the day as an argument?

Reply:
Forgot to add that, I will add it